modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf-cdi/enumClass.mustache

```diff
@@ -1,16 +1,33 @@
-import javax.xml.bind.annotation.XmlEnum;
-import javax.xml.bind.annotation.XmlType;
+@XmlType(name="{{datatypeWithEnum}}")
```

Comment:
My point is that this is a simple case of one PR, two issues.  We should get into the habit of one PR, one issue.